packages/notebook/src/celllist.ts

```diff
@@ -0,0 +1,552 @@
+// Copyright (c) Jupyter Development Team.
+// Distributed under the terms of the Modified BSD License.
+
+import {
+  ArrayExt, IIterator, IterableOrArrayLike, each, toArray, ArrayIterator
+} from '@phosphor/algorithm';
+
+import {
+  ISignal, Signal
+} from '@phosphor/signaling';
+
+import {
+  utils
```

Comment:
The uuid function is now available in `@jupyterlab/coreutils`, the one in services is going away.